Dockerfile

```diff
@@ -1,12 +1,11 @@
-FROM ruby:2.5.3
+FROM ruby:2.5.9
 
 LABEL maintainer Travis CI GmbH <support+travis-migrations-docker-images@travis-ci.com>
 
+RUN apt-key adv --fetch-keys 'https://www.postgresql.org/media/keys/ACCC4CF8.asc'
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list
```

Comment:
pls put it into one RUN (the one below) to avoid additional layers
+  'apt upgrade -y' to keep the image up-to-date

Reply:
@GbArc done